sofiadinizms/HackGrrrl
# Feature/where to buy responsive

## What I did

- The screen responsive "Where to buy"

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/where-to-buy-responsive`
   1. `git pull origin feature/where-to-buy-responsive`
   1. `cd client` or `cd server`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
